lab-taylor-melanie-ken/lib/bitmap-constructor.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const constructor = module.exports = function(data, err) {
+  if(err) return err;
```

Comment:
This isn't really an error-first function. Export the constructor directly and then use it to instantiate new objects directly where you need them.